src/main/java/com/kodilla/ecommercee/domain/Cart.java

```diff
@@ -16,7 +16,7 @@
     @Column(name = "cart_id")
     private Long id;
 
-    @NonNull
+    @NotNull
     @OneToOne
     @JoinColumn(name = "order_id")
     private Cart cart;
```

Comment:
Mając tu @NotNull przy tworzeniu kosza będzie trzeba podać Id zamówienia? Czy ta adnotacja zapewni że jak nie podamy to wpisze się powiedzmy 0 (lub inna wartość domyślna) zamiast Nulla?